app/components/post_body/post_body.js

```diff
@@ -296,7 +296,7 @@ class PostBody extends PureComponent {
                             isReplyPost={isReplyPost}
                         />
                         {this.renderFileAttachments()}
-                        {hasReactions && <Reactions postId={postId}/>}
+                        {hasReactions && <Reactions postId={postId} onAddReaction={this.props.onAddReaction} />}
```

Comment:
you have some eslint errors here, to fix do:

```js
{hasReactions &&
<Reactions
    postId={postId}
    onAddReaction={this.props.onAddReaction}
/>
}
```

Reply:
done